barretenberg/cpp/src/barretenberg/flavor/ultra_keccak_flavor.hpp

```diff
@@ -76,14 +76,11 @@ class UltraKeccakFlavor : public bb::UltraFlavor {
          */
         fr add_hash_to_transcript(const std::string& domain_separator, Transcript& transcript) const override
         {
-            // TODO(https://github.com/AztecProtocol/barretenberg/issues/1427): We need to update this function to look
-            // like UltraFlavor's add_hash_to_transcript. Alternatively, the VerificationKey class will go away when we
-            // add pairing point aggregation to the solidity verifier.
-            uint64_t circuit_size = 1 << this->log_circuit_size;
-            transcript.add_to_hash_buffer(domain_separator + "vk_log_circuit_size", circuit_size);
-            transcript.add_to_hash_buffer(domain_separator + "vk_num_public_inputs", this->num_public_inputs);
-            transcript.add_to_hash_buffer(domain_separator + "vk_pub_inputs_offset", this->pub_inputs_offset);
-            return 0;
+            // This hash contains a hash of the entire vk - including all of the elements
+            const fr hash = this->hash();
```

Comment:
this->hash includes the hashes that i removed here + the hashing of the precomputed commitments